Description: Prevented installing README on /usr/share/games/zatacka. It is fixed with debian/docs.
Author: Alexandre Dantas <alex.dantas92@gmail.com>
Last-Update: 2013-05-02
--- a/Makefile.am
+++ b/Makefile.am
@@ -6,16 +6,14 @@
 EXTRA_DIST = \
 acinclude.m4 \
 COPYING \
-LICENSE \
-README 
+LICENSE
 
 install-data-local:
 	 @$(PRE_INSTALL)
 	 $(mkinstalldirs) $(DESTDIR)$(DEFAULT_LIBDIR)/$(PACKAGE); \
 	 $(INSTALL_DATA) $(srcdir)/img/main_screen.jpg $(DESTDIR)$(DEFAULT_LIBDIR)/$(PACKAGE); \
 	 $(INSTALL_DATA) $(srcdir)/img/barrier.jpg $(DESTDIR)$(DEFAULT_LIBDIR)/$(PACKAGE); \
-	 $(INSTALL_DATA) $(srcdir)/maps/map1.jpg $(DESTDIR)$(DEFAULT_LIBDIR)/$(PACKAGE); \
-	 $(INSTALL_DATA) $(srcdir)/README $(DESTDIR)$(DEFAULT_LIBDIR)/$(PACKAGE); 
+	 $(INSTALL_DATA) $(srcdir)/maps/map1.jpg $(DESTDIR)$(DEFAULT_LIBDIR)/$(PACKAGE);
 
 uninstall-local:
 	rm -rf $(DEFAULT_LIBDIR)/zatacka/*
